py/travis-ci/u_boot_boardenv_qemu_x86_na.py

```diff
@@ -12,3 +12,11 @@
     "size": os.path.getsize(u_boot_bin),
     "crc32": hex(binascii.crc32(open(u_boot_bin, 'rb').read()) & 0xffffffff)[2:],
 }
+
+helloworld_bin = os.environ['UBOOT_TRAVIS_BUILD_DIR'] + "/lib/efi_loader/helloworld.efi"
+
+env__efi_loader_helloworld_file = {
+    "fn": "lib/efi_loader/helloworld.efi",
+    "size": os.path.getsize(helloworld_bin),
+    "crc32": hex(binascii.crc32(open(helloworld_bin, 'rb').read()) & 0xffffffff)[2:],
```

Comment:
This is used a lot now! I'd suggest implementing a utilty module/function for set the crc32 value, or perhaps even to create the whole dict object.